tools/gulp/tasks/screenshots.ts

```diff
@@ -1,15 +1,26 @@
 import {task} from 'gulp';
 import {readdirSync, statSync, existsSync, mkdirp, readFileSync, writeFileSync} from 'fs-extra';
+import {openScreenshotsBucket, connectFirebaseScreenshots} from '../util/firebase';
+import {isTravisMasterBuild} from '../util/travis-ci';
+
 import * as path from 'path';
 import * as firebaseAdmin from 'firebase-admin';
-import * as firebase from 'firebase';
-import {
-  openScreenshotsBucket,
-  connectFirebaseScreenshots} from '../util/firebase';
-import {isTravisMasterBuild} from '../util/travis-ci';
 
+// Firebase provides TypeScript definitions that are only accessible from specific namespaces.
+// This means that those types are really long and it's nearly impossible to write a function that
+// doesn't exceed the maximum columns. Import the types from the namespace so they are shorter.
+import Database = firebaseAdmin.database.Database;
+import DataSnapshot = firebaseAdmin.database.DataSnapshot;
```

Comment:
Hmm not sure. It should be pretty clear from the file name and from the comment as well.